packages/swapper/src/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -34,20 +35,19 @@ export const isArbitrumBridgeTradeQuote = (
 export const getTradeQuoteWithWallet = async (
   inputWithWallet: GetEvmTradeQuoteInputWithWallet,
   deps: SwapperDeps,
-) => {
+): Promise<Result<ArbitrumBridgeTradeQuote, SwapErrorRight>> => {
   const { wallet, ...input } = inputWithWallet
   const supportsEIP1559 = supportsETH(wallet) && (await wallet.ethSupportsEIP1559())
 
-  return getTradeQuote(
+  const quote = await getTradeQuote(
     {
       ...input,
-      sellAsset: input.sellAsset,
-      buyAsset: input.buyAsset,
-      accountNumber: input.accountNumber,
       supportsEIP1559,
     },
     deps,
   )
+
+  return quote as Result<ArbitrumBridgeTradeQuote, SwapErrorRight>
```

Comment:
nitpick - no need for the casted return that I'm seeing

Reply:
Nice catch, leftover from initial types plumbing (when quotes and rates where split at type-level): f79211f001